python/src/trezorlib/cardano.py

```diff
@@ -163,24 +169,24 @@ def _create_tokens(tokens) -> List[messages.CardanoTokenType]:
     return result
 
 
-def _create_change_output_address_parameters(
-    output,
+def _create_address_parameters_internal(
```

Comment:
I'd consider renaming the helper functions to transform the json object from `create` to `parse_xxx` and create would be only for the functions which take the individual parameters used to construct the final protobuf message. This "internal" suffix is imho too ambiguous (internal function or internal address?)

Reply:
I've renamed the functions. https://github.com/vacuumlabs/trezor-firmware/pull/57/commits/33df925901fb05ec65cef6f70e45f5aa26a96694. I don't have an issue with the reuse of the `create_address_parameters` function in `_parse_address_parameters` though.